images/oidc-discovery-provider/README.md

```diff
@@ -0,0 +1,24 @@
+<!--monopod:start-->
+# oidc-discovery-provider
+| | |
+| - | - |
+| **Status** | stable |
+| **OCI Reference** | `cgr.dev/chainguard/oidc-discovery-provider` |
+| **Variants/Tags** | ![](https://storage.googleapis.com/chainguard-images-build-outputs/summary/oidc-discovery-provider.svg) |
+
+*[Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+Minimalist Wolfi-based `oidc-discovery-provider` image. The SPIRE OIDC Discovery Provider is a small helper that provides a minimal implementation of a subset of the OIDC discovery document as related to exposing a JSON Web Key Set (JWKS) for JSON Web Token (JWT) validation.
+
+> _https://github.com/spiffe/spire/tree/main/support/oidc-discovery-provider_
+
+## Get It!
+
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/oidc-discovery-provider:latest
```

Comment:
Shouldn't this have been 

cgr.dev/chainguard/spire-oidc-discovery-provider:latest

Reply:
Indeed! It looks like we'll be fixing this soon as part of #1019